### ISSUE TITLE
Node failed to join the cluster when I use the experimental feature “consistent-topology-changes”

### ISSUE THREAD

**Issue by xxuvincent** [scylladb/scylladb#16869]
*Installation details*
Scylla version (or git commit hash): 5.4.0
Cluster size: 12

Now I am starting a cluster with experimental feature “consistent-topology-changes” . For the first eleven nodes , I enabled the experimental feature successfully . However , when it comes to the twelve node , it failed to join the cluster . I am confused why does this situation happen ?

Here are error logs of seed node and failed-to-join node:

seed node:
```bash
`Jan 17 16:21:34 HOSTNAME-474 scylla[285367]:  [shard  0:goss] storage_service - raft topology: received request to join from host_id: 4e3aabb9-6c7a-47eb-8387-add7530c239f
Jan 17 16:21:34 HOSTNAME-474 scylla[285367]:  [shard  0:comp] compaction - [Compact system.topology 6d233740-b511-11ee-bf7b-577b28854eb1] Compacting [/var/lib/scylla/data/system/topology-5be1feb3929e3df98da8d771c36129a0/me-3gcu_0n7y_31qm82wttuuo4qd701-big-Data.db:level=0:origin=memtable,/var/lib/scylla/data/system/topology-5be1feb3929e3df98da8d771c36129a0/me-3gcu_0n4z_4htm82wttuuo4qd701-big-Data.db:level=0:origin=compaction]
Jan 17 16:21:34 HOSTNAME-474 scylla[285367]:  [shard  0:comp] compaction - [Compact system.topology 6d233740-b511-11ee-bf7b-577b28854eb1] Compacted 2 sstables to [/var/lib/scylla/data/system/topology-5be1feb3929e3df98da8d771c36129a0/me-3gcu_0n7y_338mo2wttuuo4qd701-big-Data.db:level=0]. 84kB to 77kB (~91% of original) in 20ms = 4MB/s. ~256 total partitions merged to 1.
Jan 17 16:21:34 HOSTNAME-474 scylla[285367]:  [shard  0:goss] storage_service - raft topology: placed join request for 4e3aabb9-6c7a-47eb-8387-add7530c239f
Jan 17 16:21:34 HOSTNAME-474 scylla[285367]:  [shard  0:stre] storage_service - raft topology: coordinator fiber found a node to work on id=4e3aabb9-6c7a-47eb-8387-add7530c239f state=none
Jan 17 16:21:34 HOSTNAME-474 scylla[285367]:  [shard  0:stre] boot_strapper - Get random bootstrap_tokens={8336017245321990027, 973663884872274533, -5164012693825548672, 5019371039235828931, 858665839054710112, -1204105103958564571, 30737206198329322, -6693601071127569229, 675513877036375421, -5291956843044808537, -2302961623029350256, 8657594883213710164, 6392093461365975013, -416248984177633267, -2344598757689963218, -6683049544548644048, 1596859709471683206, -8544609107597968625, 2908583811465935054, 2818002839357975008, 1289397113165804962, 3515556408317177263, 7117626809434027574, -488964047458314082, 2468123419696946774, 2749899316402834836, -1650391404609323158, 6610696515484634294, 7279906877390806888, 309532215507735449, -3015777836207378, 1585262794704690378, -930297647271549828, 6126195858737975296, -8677098108663701671, -2937863045246194440, 1407251562221075461, -3217534022137946944, -4240879872575670619, -2826965453973843933, 572065232538281203, -2027806326337950993, -7576367212542394984, 6311993433939534591, 1185491637338811801, -2056934790554175505, 1508324244430326020, 6096832331269649051, 7572353160530675969, -4801926724821412711, -668927735813903369, -3944513774038692172, -1829893337417474773, -1609400635372606819, 246636428396449557, 5289476507939217069, 6693720188682965130, 2564920424209021337, 5422794725502417876, -3809090170653740700, -554604473293730120, -8300427663064666217, -4798860978944530481, -2395311961563163251, 2028918542592901250, 5774846644669337125, -535609541873207545, 6261702226790385388, 3863540592255784506, 2474440631584904386, 2173824618913684584, -6138114488400079510, -1599449605755402634, -7361451296441746619, -512870142762614378, 6180547450221385886, -1157070813176623658, 4459379329125228818, -5699366425444912499, -9207948207560270393, -4773228495140959931, 2046473617909595549, -7406613162251078307, -2343255096721443430, 410772918152445534, -8214206509869147966, -1496239593655146229, 2606738339752785551, 7296216634374726081, 8924356206228955993, -2646059326454403366, -2869758167218285231, -6067779390349086746, -3042760528842511833, -6198297881506537051, -7590747844099278005, -8132400578950258457, 6315335118592140956, -4623815773154262739, -7747846459470230500, 501483743170297131, -3625543459397136455, -4521130006717109046, 8030928244759439430, -1142187138228610393, -8909318319273127095, -3344556416917702019, 1855297007195530717, 6660490421658847769, -3370748464176825320, -1413234515663237920, 6480504983418670498, 3176820168789912946, 6706648289588346269, 1398531241491047817, 8876070541162979320, 3170788190059640053, -4517041049889337300, 690771930033394694, -4190336271845763973, 733396380174825277, -3659024928506865393, 2909686192168541674, 5368975963241420640, 3504188288248447055, -5125881811072273177, -3783124091982300677, 1199247728518902038, 875462142331296409, -3277205874193948108, -3865693729897339182, 7551919423134080247, 5677708211207723739, 6308775793208094412, -2647203946343420043, -3372935573128314900, 4050493902808705206, -8946528237867081310, -6182943349904166300, 5335422701044047459, 7317627780832566541, 2461329134140837047, -8012375105721874542, -2886796087918355817, -1471429299727578030, 2911024716315977886, 5331219926692249907, -4520817165293600559, -5998795718628251818, 5349475544574252307, -1369816647856891086, -5278484488214587278, 8817091865745732467, -7236122912296931917, -9204616377787242845, 8850385851623396340, 8331325513034499846, 1331393741549767383, 4947478655461236945, 461772884203636016, -2451882316154740641, 5018298929699607799, 6020629660680563715, -8405428950390380271, 80349882661796868, 944597339536611370, -180211194225181254, -3285116252243073666, -8776971568637417575, -1512240789183976570, -8644813667044044422, 1508884465021051001, 6851163273864114702, -1901926470519273638, 339656404816551530, 8148115744598988107, 4040026367120014114, -3223077702215183085, 7302258949024679263, 2706919280988879254, 4784235594613770688, 2108299022075007715, 3717177303732528556, 7899262183047350026, -3428412094069088610, -1874219937220579405, 5710370666594637119, -613014797683553740, 1507418216136655355, -8199617076843202949, 2224065501537218185, -5462679413892978179, -5291351963034662968, -3781330132381259972, -2687275703498726879, 35428229810653207, -8477878322767498667, 943209931836363519, 1068387451926837712, -9135644725626443358, 3860418867484593925, -4933307392313283421, -5195425510108792316, -6776839790726279385, -5345687076747114495, 8050581962746445168, 9137555215044834099, -6919039999296127841, 8577900516475357795, 5356640630443613344, -8988095469240431863, 5081943975423978097, -3633729149151837066, 4487660398252650924, -7377216483818259101, -4584744557935046744, 2432821192328040950, 600118722452697087, -52397582968474253, -4662120339301675515, -6980115554285814220, -115232855664478252, -2804262953216017406, 6320890440269349616, -8141296578102344219, -4931108371176781050, 7061919376359527780, -278976458065063814, -5872161224707838486, -6828008417168081193, -4979175190914077046, 8427650561372001793, 7994507448829495466, 3074351370766077621, -7260722201419990148, 5415485819380202170, 4049581908758776107, 8447881551103312526, 8147365974880779081, -8109026374410690360, 6690619926031301831, 1002725080784309273, 2587027835411284328, -4271583981496867050, 4136569743238570759, 8097584524209193006, 6589671088025675329, 195735135696073519, -8549119923709762209, 1433331746181601992, -5562844122173147655, -859574556018679036, 5523729271892512857, 273827428441362092, 8052687447353216671, 3821124290743261401}
Jan 17 16:21:34 HOSTNAME-474 scylla[285367]:  [shard  0:stre] storage_service - (rate limiting dropped 184 similar messages) raft topology: cannot map 4e3aabb9-6c7a-47eb-8387-add7530c239f to ip, retrying.
Jan 17 16:21:35 HOSTNAME-474 scylla[285367]:  [shard  0:goss] gossip - InetAddress 10.249.138.164 is now UP, status = UNKNOWN
Jan 17 16:21:35 HOSTNAME-474 scylla[285367]:  [shard  0:main] raft_group_registry - marking Raft server 4e3aabb9-6c7a-47eb-8387-add7530c239f as alive for raft groups
Jan 17 16:21:35 HOSTNAME-474 scylla[285367]:  [shard  0:stat] mutation_partition - Memory usage of unpaged query exceeds soft limit of 1048576 (configured via max_memory_for_unlimited_query_soft_limit)
Jan 17 16:21:35 HOSTNAME-474 scylla[285367]:  [shard  0:stre] raft - [f84f4024-9e66-4702-96b9-502e262fe97b] Transferring snapshot to 4e3aabb9-6c7a-47eb-8387-add7530c239f failed with: seastar::rpc::remote_verb_error (Mutation of 19546409 bytes is too large for the maximum size of 16777216)
Jan 17 16:21:35 HOSTNAME-474 scylla[285367]:  [shard  0:stat] mutation_partition - Memory usage of unpaged query exceeds soft limit of 1048576 (configured via max_memory_for_unlimited_query_soft_limit)
Jan 17 16:21:35 HOSTNAME-474 scylla[285367]:  [shard  0:stre] raft - [f84f4024-9e66-4702-96b9-502e262fe97b] Transferring snapshot to 4e3aabb9-6c7a-47eb-8387-add7530c239f failed with: seastar::rpc::remote_verb_error (Mutation of 19546409 bytes is too large for the maximum size of 16777216)
Jan 17 16:21:36 HOSTNAME-474 scylla[285367]:  [shard  0:stat] mutation_partition - Memory usage of unpaged query exceeds soft limit of 1048576 (configured via max_memory_for_unlimited_query_soft_limit)
Jan 17 16:21:36 HOSTNAME-474 scylla[285367]:  [shard  0:stre] raft - [f84f4024-9e66-4702-96b9-502e262fe97b] Transferring snapshot to 4e3aabb9-6c7a-47eb-8387-add7530c239f failed with: seastar::rpc::remote_verb_error (Mutation of 19546409 bytes is too large for the maximum size of 16777216)
Jan 17 16:21:36 HOSTNAME-474 scylla[285367]:  [shard  0:stat] mutation_partition - Memory usage of unpaged query exceeds soft limit of 1048576 (configured via max_memory_for_unlimited_query_soft_limit)
Jan 17 16:21:36 HOSTNAME-474 scylla[285367]:  [shard  0:stre] raft - [f84f4024-9e66-4702-96b9-502e262fe97b] Transferring snapshot to 4e3aabb9-6c7a-47eb-8387-add7530c239f failed with: seastar::rpc::remote_verb_error (Mutation of 19546409 bytes is too large for the maximum size of 16777216)
Jan 17 16:21:36 HOSTNAME-474 scylla[285367]:  [shard  0:stat] mutation_partition - Memory usage of unpaged query exceeds soft limit of 1048576 (configured via max_memory_for_unlimited_query_soft_limit)
Jan 17 16:21:36 HOSTNAME-474 scylla[285367]:  [shard  0:stre] raft - [f84f4024-9e66-4702-96b9-502e262fe97b] Transferring snapshot to 4e3aabb9-6c7a-47eb-8387-add7530c239f failed with: seastar::rpc::remote_verb_error (Mutation of 19546409 bytes is too large for the maximum size of 16777216)
Jan 17 16:21:36 HOSTNAME-474 scylla[285367]:  [shard  0:stat] mutation_partition - Memory usage of unpaged query exceeds soft limit of 1048576 (configured via max_memory_for_unlimited_query_soft_limit)
Jan 17 16:21:36 HOSTNAME-474 scylla[285367]:  [shard  0:stre] raft - [f84f4024-9e66-4702-96b9-502e262fe97b] Transferring snapshot to 4e3aabb9-6c7a-47eb-8387-add7530c239f failed with: seastar::rpc::remote_verb_error (Mutation of 19546409 bytes is too large for the maximum size of 16777216)
```

failed-to-join node:
```bash
`Jan 17 16:24:32 HOSTNAME-485 scylla[158302]: [shard 0:stat] schema_tables - Schema version changed to 187398f8-6575-3b4d-b98b-5602749c6ec3
Jan 17 16:24:33 HOSTNAME-485 scylla[158302]: [shard 0:stat] schema_tables - Schema version changed to 187398f8-6575-3b4d-b98b-5602749c6ec3
Jan 17 16:24:33 HOSTNAME-485 scylla[158302]: [shard 0:stat] schema_tables - Schema version changed to 187398f8-6575-3b4d-b98b-5602749c6ec3
Jan 17 16:24:33 HOSTNAME-485 scylla[158302]: [shard 0:stat] schema_tables - Schema version changed to 187398f8-6575-3b4d-b98b-5602749c6ec3
Jan 17 16:24:33 HOSTNAME-485 scylla[158302]: [shard 0:stat] schema_tables - Schema version changed to 187398f8-6575-3b4d-b98b-5602749c6ec3
Jan 17 16:24:33 HOSTNAME-485 scylla[158302]: [shard 0:stat] schema_tables - Schema version changed to 187398f8-6575-3b4d-b98b-5602749c6ec3
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:stat] schema_tables - Schema version changed to 187398f8-6575-3b4d-b98b-5602749c6ec3
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:stat] schema_tables - Schema version changed to 187398f8-6575-3b4d-b98b-5602749c6ec3
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down group 0 service
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down group 0 service was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down storage service notifications
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down storage service notifications was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down system distributed keyspace
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down system distributed keyspace was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down migration manager notifications
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down migration manager notifications was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down storage service notifications
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down storage service notifications was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down sstables loader API
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down sstables loader API was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down sstables loader
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down sstables loader was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down repair API
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down repair API was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down messaging service API
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down messaging service API was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down storage service messaging
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down storage service messaging was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down cdc log service
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down cdc log service was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down CDC Generation Management service
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down CDC Generation Management service was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down repair service
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] task_manager - Stopping module repair
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] task_manager - Unregistered module repair
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 2:main] task_manager - Stopping module repair
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 6:main] task_manager - Stopping module repair
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 7:main] task_manager - Stopping module repair
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 13:main] task_manager - Stopping module repair
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 17:main] task_manager - Stopping module repair
………………
………………
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down storage_service was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down tablet allocator
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down tablet allocator was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down direct_failure_detector
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down direct_failure_detector was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down fd_pinger
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down fd_pinger was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down raft_address_map
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down raft_address_map was successful
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down gossiper
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] gossip - My status = UNKNOWN
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] gossip - No local state or state is in silent shutdown, not announcing shutdown
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:main] gossip - Disable and wait for gossip loop started
Jan 17 16:24:34 HOSTNAME-485 scylla[158302]: [shard 0:goss] rpc - client 10.249.138.159:50784 msg_id 180: exception “gate closed” in no_wait handler ignored
Jan 17 16:24:35 HOSTNAME-485 scylla[158302]: [shard 0:goss] rpc - client 10.249.138.157:51552 msg_id 181: exception “gate closed” in no_wait handler ignored
Jan 17 16:24:35 HOSTNAME-485 scylla[158302]: [shard 0:goss] rpc - client 10.249.138.154:63312 msg_id 183: exception “gate closed” in no_wait handler ignored
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:goss] rpc - client 10.249.138.158:50496 msg_id 183: exception “gate closed” in no_wait handler ignored
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:stre] gossip - failure_detector_loop: Finished main loop
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] gossip - Gossip is now stopped
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down gossiper was successful
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down messaging service
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Shutting down nontls server
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Shutting down tls server
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Shutting down tls server - Done
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.156:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.158:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.155:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.162:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.160:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.157:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.154:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.161:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.159:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.163:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.153:0
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopped clients
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] messaging_service - Stopping client for address: 10.249.138.153:0
…………
…………
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down compaction_manager was successful
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down task_manager
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down task_manager was successful
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down service_memory_limiter
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down service_memory_limiter was successful
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down sst_dir_semaphore
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down sst_dir_semaphore was successful
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down migration manager notifier
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down migration manager notifier was successful
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down prometheus API server
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down prometheus API server was successful
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down API server
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down API server was successful
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down sighup
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down sighup was successful
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down configurables
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Shutting down configurables was successful
Jan 17 16:24:36 HOSTNAME-485 scylla[158302]: [shard 0:main] init - Startup failed: seastar::timed_out_error (timedout)`
```

**Comment by xxuvincent** [comment]
I referred to the discussions in issue #13864 and tried to increase the limit of 'Commit log segment size'. 
When I changed the limit from 32MB to 64MB , the cluster expanded from 12 nodes to 18 nodes. However , I plan to deploy a cluster with 180 nodes. In that case, I speculate that the limit of 'Commit log segment size' may need to be set to a very large value. 
I'm not sure whether simply increasing the limit of 'Commit log segment size' will have an impact on the performance and stability of the cluster.
